manifests/cf-manifest/manifest/010-cf-jobs.yml

```diff
@@ -222,6 +224,9 @@ jobs:
       - name: cf
     update:
       serial: true
+    properties:
+      bbs:
+        url: (( grab meta.bbs.api_location  ))
```

Comment:
The metric explorer showed an absence of datapoints for that period of time too. I haven't been able to reproduce it since, but I now understand that it's not related to both machines submitting.

Piotr and I used `iptables` to prevent one machine from connecting to the active BBS. The monitor triggered successfully, because `min()` selected the `0` metric from the non-active machine rather than the `1` metric from the active machine.